src/main/resources/application.yml

```diff
@@ -40,8 +40,12 @@ app:
     oauth-client:
       resource: "${app.indexing-service.base-uri}"
     publisher-catalog-name: publisher_data
-  query-cleanup-interval: "*/10 * * ? * *" # Every 10 seconds
-  query-cleanup-timeout-in-seconds: 120
+  query-cleanup:
+    cron-interval: "*/10 * * ? * *" # Every 10 seconds
+    query-timeout-in-seconds: 120
+  query-job-cleanup:
+    cron-interval: "0 0 1 * * ?" # Every day at 1am
+    query-deletion-timeout-in-days: 1
```

Comment:
Same like above, how about we use `timeout-in-seconds` as the key here?

Reply:
Done